solidity/test/token_stake/TestTopUps.js

```diff
@@ -272,6 +265,18 @@ describe('TokenStaking/TopUps', () => {
       )   
     })
 
+    it("can be done in one step during initialization period", async () => {
+      // half of the initialization period passed
+      await time.increase(initializationPeriod.divn(2))
+
+      await initiateTopUp()
+
+      const delegationInfo = await tokenStaking.getDelegationInfo(operatorOne)
+      expect(delegationInfo.amount).to.eq.BN(delegatedAmount.muln(2))
+      expect(delegationInfo.createdAt).to.eq.BN(await time.latest())
+      expect(delegationInfo.undelegatedAt).to.eq.BN(0)
+    })
```

Comment:
Thoughts on adding a case where we combine top ups during and after the initialization period?
Steps:
1. top up during the initialization period
2. increase time to finish the initialization period
3. top up after the initialization period
4. commit top up
5. validations

Reply:
Sure, I have even moved one step further and made sure we can do one yet top-up after the initialization period. Please see 301826b.